packages/onramp-kit/src/packs/monerium/MoneriumPack.ts

```diff
@@ -156,18 +100,11 @@ export class MoneriumPack extends OnRampKitBasePack {
       }
     }
 
-    const authFlowUrl = this.client.getAuthFlowURI({
-      client_id: this.#config.clientId,
-      redirect_uri: redirectUrl,
+    await this.client.authorize({
       address: safeAddress,
       signature: '0x',
-      chain: await this.client.getChain(),
-      network: await this.client.getNetwork()
+      chainId: await this.client.getChainId()
```

Comment:
I would suggest that you align this in your SDK. For some methods, like `linkAddress`, you use `chain` and `network`. This one now uses directly `chainId`.